source/content/http-to-https.md

```diff
@@ -114,12 +121,17 @@ Use the following techniques to replace insecure references to your domain in th
 </TabList>
 
 ## Clear Caches
+
 Clear Drupal and WordPress object caches in the [database and/or in Redis](/redis/#clear-cache) in addition to manually flush edge caches by going to your Pantheon Dashboard and clicking the **Clear Caches** button.
 
 At this point, all visitors to the site should be able to securely access all pages over HTTPS with no browser warnings.
 
 ## Redirect to HTTPS and the primary domain
 
+The best way to redirect to HTTPS is via the [HSTS header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security). Pantheon lets you configure this header in the [pantheon.yml](/pantheon-yml) file.
```

Comment:
@davidneedham this is the section of changed content that should resolve the issue you filed.